std/path.d

```diff
@@ -84,13 +102,24 @@ else static assert (0, "unsupported platform");
     On Windows, this includes both $(D `\`) and $(D `/`).
     On POSIX, it's just $(D `/`).
 */
-bool isDirSeparator(dchar c)  @safe pure nothrow @nogc
+bool isDirSeparator(PathStyle ps = pathStyle)(dchar c)  @safe pure nothrow @nogc
```

Comment:
This is a breaking change, because code that took the address of isDirSeparator now has to supply an template argument.

bool isDirSeparator(dchar c, PathStyle ps = pathStyle)